src/components/List.js

```diff
@@ -51,6 +54,48 @@ const List = ({ firestore }) => {
     setInputText('');
   };
 
+  //* Created two lists
+  const inactiveList = [];
+  const activeList = [];
+
+  //* Used this function to divide active and inactive items
+  filteredList.map(item => {
```

Comment:
`.map` returns an array. I think what you want here is `.forEach` since you're pushing items into new arrays more manually.

We can talk about the difference on our call if it's helpful!